apps/app/messages/en.json

```diff
@@ -306,8 +322,10 @@
       "invalidVaultList": "No valid vault list found",
       "notEnoughTokens": "Not enough {symbol} in wallet",
       "invalidNumber": "Enter a valid number",
+      "invalidAddress": "Enter a valid EVM address",
+      "sameAsDelegate": "Entered address is same as current delegate",

```

Comment:
Don't think we should have a custom error for this. We should just disable the button if that's the case. Otherwise it's quite possible the initial form would always be in this state prior to validation.

Reply:
The initial form state is empty, however the placeholder says the current value (by default, your address).

I thought about leaving the button disabled and not having a validation for this, however I'm trying to cover the case where someone enters the wrong address by accident, and having the form tell them where they messed up instead of them being confused as to why they can't click the button to send the transaction.